single-line-diagram-viewer/src/main/java/com/powsybl/sld/viewer/SingleLineDiagramViewer.java

```diff
@@ -279,13 +282,42 @@ private ContainerDiagramResult createContainerDiagramView(WebView diagramView, C
             }
 
             try (InputStream metadataInputStream = new ByteArrayInputStream(metadataData.getBytes(StandardCharsets.UTF_8))) {
-                ContainerDiagramView.loadSvgAndMetadata(diagramView, svgData, metadataInputStream, switchId -> handleSwitchPositionchange(c, switchId), SingleLineDiagramViewer.this);
+                loadSvgAndMetadata(diagramView, svgData, metadataInputStream, switchId -> handleSwitchPositionchange(c, switchId), SingleLineDiagramViewer.this);
             } catch (IOException e) {
                 throw new UncheckedIOException(e);
             }
             return new ContainerDiagramResult(svgData, metadataData, jsonData);
         }
 
+        protected void loadSvgAndMetadata(WebView diagramView, String svgInputStream,
+                                                 InputStream metadataInputStream,
+                                                 SwitchPositionChangeListener listener,
+                                                 DisplayVoltageLevel displayVL) {
+            // convert svg file to WebView components
+            try {
+                loadContent(diagramView, svgInputStream);
+
+                // load metadata
+                GraphMetadata metadata = GraphMetadata.parseJson(metadataInputStream);
+
+                // install node and wire handlers to allow diagram edition
+                // installHandlers(svgImage, metadata, listener, displayVL);  FIXME: add handlers with js
+            } catch (Exception e) {
+                // to feed the content of the 'SVG' and 'Metadata' tab, even if the
+                // svg diagram cannot be loaded by svg loader, or if the handlers cannot be installed
+                LOGGER.warn("Error in loading svg image or installing handlers : {}", e.getMessage());
+            }
+        }
+
+        public void loadContent(WebView diagramView, String svgInputStream) {
+            String b = "<html>" +
+                    String.format("<body><div id='%1$s'>", divId) +
```

Comment:
There seems to be a margin of a few pixel when adding body or div, could you try removing it by specifying a 0 margin in body / div tag?